src/main/java/Duke/Duke.java

```diff
@@ -0,0 +1,49 @@
+package Duke;
+
+import java.util.Scanner;
+
+public class Duke {
+    private final static String LINE_BREAK = " ____________________________________________________________";
```

Comment:
Good job using constants for your strings! You could extend that to more magic strings (e.g `"bye"`) as well as magic numbers!

Reply:
You could also extend it to your booleans where possible!